tests/Oryx.Tests.Common/TestTempDirTestFixture.cs

```diff
@@ -18,6 +18,8 @@ public TestTempDirTestFixture()
             RootDirPath = Path.Combine(Path.GetTempPath(), "oryxtests", Guid.NewGuid().ToString());
 
             Directory.CreateDirectory(RootDirPath);
+
+            Environment.SetEnvironmentVariable("DEBIAN_FLAVOR", "stretch");
```

Comment:
nit: would you mind leaving a comment that mentions why we're setting this to stretch? (I assume this is just the default value we want to use, but just want the reader to be sure).

Reply:
Yeah the flavor here doesn't really matter, we just need the CLI to find _some_ debian flavor so it doesn't fail. Could also have provided an .ostype file for it to parse. I'll add a comment for that